Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.h

```diff
@@ -56,7 +56,7 @@ namespace AzToolsFramework
             /// ComponentMode, add them too (duplicates will not be added - handled by AddComponentMode)
             void AddOtherSelectedEntityModes();
 
-            /// Return is the Editor-wide ComponentMode state active.
+            /// Return if the Editor-wide ComponentMode state active.
```

Comment:
Nit: "state active" -> "state is active". If we're improving the grammar here, may as well take it all the way. :)